test/sharness/Makefile

```diff
@@ -14,6 +14,9 @@ IPFS_ROOT = ../..
 # User might want to override those on the command line
 GOFLAGS =
 
+# The targets in this makefile depend on being run sequentially
+.NOTPARALLEL:
```

Comment:
@jbenet about Git I had a look and yeah some work has been done a long time ago so that the tests can run in parallel (see https://github.com/git/git/commit/abc5d372ec242fc654dc6780df6ea3d63dc72f2f).

And there are always some flags or options or other stuff in the IPFS-BUILD-OPTIONS like files, so that there is no problem with empty files.